ERCS/erc-z.md

```diff
@@ -0,0 +1,128 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7763
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
Okay